var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -167,29 +167,21 @@ class Cmake(Package):
                   'See: https://gitlab.kitware.com/cmake/cmake/-/issues/21135')
     conflicts('%nvhpc')
 
-    # Really this should conflict since it's enabling or disabling openssl for
-    # CMake's internal copy of curl.  Ideally we'd want a way to have the
-    # openssl variant disabled when ~ownlibs but there's not really a way to
-    # tie the values of those togethor, so for now we're just going to ignore
-    # the openssl variant entirely when ~ownlibs
-    # conflicts('~ownlibs', when='+openssl')
-
-    depends_on('curl',           when='~ownlibs')
-    depends_on('expat',          when='~ownlibs')
```

Comment:
CMake has a direct dependency on expat so that shouldn't be removed

Reply:
Thanks, fixed that